### PR TITLE
fix(comp:input): update style with selector

### PR DESCRIPTION
**Pull request by danranVm**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [our guidelines](https://github.com/IDuxFE/idux/blob/main/packages/site/src/docs/Contributing.zh.md#commit)
- [ ] Tests for the changes have been added/updated or not needed
- [ ] Docs and demo have been added/updated or not needed

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Component style update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Application (the showcase website) / infrastructure changes
- [ ] Other... Please describe:

### CODE REVIEW

File: packages/components/form/style/status.less [hunk 1]
```diff
@@ -77,13 +77,13 @@
         color: @form-item-invalid-color;
         border-color: @form-item-invalid-color;
 
-        .@{idux-prefix}-select {
-          &:hover .@{idux-prefix}-select-selector,
-          .@{idux-prefix}-select-selector {
+        .@{idux-prefix}-selector {
```
Comment (liuzaijiang) [inline packages/components/form/style/status.less:80]: .@{idux-prefix}-selector 改成 .@{selector-prefix}

File: packages/components/_private/input/src/Input.tsx [hunk 2]
```diff
@@ -46,12 +46,12 @@ export default defineComponent({
       const { clearable, clearIcon, clearVisible, disabled, addonAfter, addonBefore, prefix, suffix, onClear } = props
       const prefixCls = mergedPrefixCls.value
 
-      const addonBeforeNode = renderAddon(slots.addonBefore, addonBefore, `${prefixCls}-addon`)
-      const addonAfterNode = renderAddon(slots.addonAfter, addonAfter, `${prefixCls}-addon`)
-      const prefixNode = renderIcon(slots.prefix, prefix, `${prefixCls}-prefix`)
-      const suffixNode = renderIcon(slots.suffix, suffix, `${prefixCls}-suffix`)
+      const addonBeforeNode = renderAddon(slots.addonBefore, addonBefore, `${prefixCls}-addon`, 'before-addon')
+      const addonAfterNode = renderAddon(slots.addonAfter, addonAfter, `${prefixCls}-addon`, 'before-addon')
```
Comment (liuzaijiang) [inline packages/components/_private/input/src/Input.tsx:50]: 'before-addon' key 重复了

File: packages/components/select/src/content/Content.tsx [hunk 3]
```diff
@@ -55,15 +55,18 @@ export default defineComponent({
       const children = [<ListBox />]
 
       if (props.searchable === 'overlay') {
+        const value = inputValue.value
         children.push(
           <div class={`${mergedPrefixCls.value}-overlay-search-wrapper`}>
             <ɵInput
               clearable
+              clearIcon="close"
```
Comment (liuzaijiang) [inline packages/components/select/src/content/Content.tsx:63]: 规范是用close-circle图标

Comment (danranVm) [reply packages/components/select/src/content/Content.tsx:63]: tree-select 也一起改了

**Pull request merged**